account-integrations/safe/test/SafeWebAuthnPlugin.t.sol

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: MIT
```

Comment:
hmm I'm inclined to think we may want to put these tests into their own folder.  Maybe a `unit` test folder.  Or `forge-tests`.  I don't have a strong opinion but I wonder if we should be more explicit these are different then the other tests

Reply:
Agreed. I'm actually leaning towards your second suggestion in your PR. Will update to that:

`test/hardhat`
`test/hardhat/integration`
`test/forge`